### ISSUE TITLE
Warning message while building fyne application

### ISSUE THREAD

**Issue by rythmkraze** [fyne-io/fyne#221]
How to eliminate this warning message that I get every time I `go build` any fyne application.
```
# fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw
In file included from ../go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/c_glfw_linbsd.go:24:0:
../go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/glfw/src/linux_joystick.c: In function ‘_glfwInitJoysticksLinux’:
../go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/glfw/src/linux_joystick.c:224:42: warning: ‘%s’ directive output may be truncated writing up to 255 bytes into a region of size 9 [-Wformat-truncation=]
             snprintf(path, sizeof(path), "%s/%s", dirname, entry->d_name);
                                          ^~~~~~~
In file included from /usr/include/stdio.h:862:0,
                 from /usr/include/X11/Xcursor/Xcursor.h:26,
                 from ../go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/glfw/src/x11_platform.h:39,
                 from ../go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/glfw/src/internal.h:169,
                 from ../go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/glfw/src/x11_init.c:28,
                 from ../go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/c_glfw_linbsd.go:19:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:64:10: note: ‘__builtin___snprintf_chk’ output between 12 and 267 bytes into a destination of size 20
   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        __bos (__s), __fmt, __va_arg_pack ());
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

```
GOARCH="amd64"
GOOS="linux"
GCCGO="gccgo"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build098331038=/tmp/go-build -gno-record-gcc-switches"
```

**Comment by andydotxyz** [comment]
Can you confirm the operating system and version you are compiling on?

**Comment by rythmkraze** [comment]
elementary OS 5.0 Juno(64-bit)
Built on Ubuntu 18.04.2 LTS
Linux 4.15.0-47-generic
GTK+ 3.22.30

`go version go1.12.1 linux/amd64`

**Comment by andydotxyz** [comment]
Thanks, is the gcc toolchain Elementary OS or Ubuntu? (I don't know how that setup works).

**Comment by rythmkraze** [comment]
elementary OS is based on Ubuntu. So gcc toolchain should be the same as the one in Ubuntu.

**Comment by andydotxyz** [comment]
It looks like this is upstream issue glfw/glfw#1025 which has been resolved in the GLFW 3.3 release. We are hope to upgrade soon - being tracked in #183

**Comment by rythmkraze** [comment]
awesome. Thanks
